app/src/main/res/layout/recyclerview_row.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+        <TextView
+            android:id="@+id/departureTimeLabel"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:text="@string/departureTimeLabel"
+            style="@style/Text.Subtitle"/>
+        <TextView
+            android:id="@+id/departureTime"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:text="@string/placeholder"
```

Comment:
Can we delete these lines with 'placeholder'? We never want to show them to the user. What can be nice is using `tools:text="22:05"`. This fns that just when in the IDE looking in design view it will populate with that value, to show the intended appearance.